app/screens/global_drafts/components/tabbed_contents/draftTabsHeader.tsx

```diff
@@ -156,6 +157,7 @@ export function DraftTabsHeader({draftsCount, scheduledPostCount, selectedTab, o
                 <TouchableOpacity
                     onPress={onScheduledPostTabPress}
                     testID='scheduled_post_tab'
+                    accessibilityState={{selected: selectedTab === DRAFT_SCREEN_TAB_SCHEDULED_POSTS}}
```

Comment:
This is interesting.. care to explain?

Reply:
This is merely added for testing. The `accessibilityState` prop provides additional context to accessibility tools (like testing tools) about a component's current state. Without this, screen readers might not be able to correctly detect which tab is currently selected.